schedulers/terraform/managed-airflow-mwaa/addons.tf

```diff
@@ -0,0 +1,87 @@
+#------------------------------------------------------------------------
+# EKS Addons
+#------------------------------------------------------------------------
+module "eks_blueprints_kubernetes_addons" {
```

Comment:
replace this name from `eks_blueprints_kubernetes_addons ` to `eks_blueprints_addons `